chromium_src/chrome/renderer/chrome_content_renderer_client.cc

```diff
@@ -6,9 +6,16 @@
 #include "brave/components/content_settings/renderer/brave_content_settings_agent_impl.h"
 #include "brave/renderer/brave_url_loader_throttle_provider.h"
 #include "chrome/renderer/url_loader_throttle_provider_impl.h"
+#include "components/feed/content/renderer/rss_link_reader.h"
 
 #define URLLoaderThrottleProviderImpl BraveURLLoaderThrottleProvider
 
+// We need to do this here rather than in |BraveContentRendererClient| because
+// it needs access to the registry on ChromeRenderFrameObserver.
+#define BRAVE_RENDER_FRAME_CREATED \
```

Comment:
I don't understand, this is already in there and the flag defaults to true. On android we override it, but we can change that.
```
// Feed is not used in Brave
    args.enable_feed_v2 = false
```
```
#if BUILDFLAG(ENABLE_FEED_V2)
  if (render_frame->IsMainFrame() &&
      base::FeatureList::IsEnabled(feed::kWebFeed)) {
    new feed::RssLinkReader(render_frame, registry);
  }
#endif
```

Reply:
actually looks like we would need to enable `kWebFeed` since that is disabled by default

Reply:
Yeah, so my idea was that we should be able to enable this feature without also needing to turn the `kWebFeed` flag on, because I think that flag adds some additional stuff which we aren't interested in turning on.

The way this is working, we can pull in `RssLinkReader` without worrying about the `ENABLE_FEED_V2` flag or whether `kWebFeed` is set, neither of which is really related to this feature.

I'm happy to rethink the approach here if you have an alternative suggestion, but I'm not sure enabling `kWebFeed` is the way we should go here.

Reply:
there's also no reason I can see for this to be a chromium_src override, this should go in `BraveContentRendererClient::ExposeInterfacesToBrowser`